plyer/platforms/ios/inappbrowser.py

```diff
@@ -0,0 +1,27 @@
+try:
+    from urllib.parse import quote
+except ImportError:
+    from urllib import quote
+
+from plyer.facades import InAppBrowser
+from pyobjus import autoclass, objc_str
+from pyobjus.dylib_manager import load_framework
+
+load_framework('/System/Library/Frameworks/UIKit.framework')
+
+NSURL = autoclass('NSURL')
+NSString = autoclass('NSString')
+UIApplication = autoclass('UIApplication')
+
+
+class iOSInAppBrowser(InAppBrowser):
+    def _open_url(self, **kwargs):
+        url = kwargs.get('url', '')
+
+        nsurl = NSURL.alloc().initWithString_(objc_str(str(url)))
+
+        UIApplication.sharedApplication().openURL_(nsurl)
```

Comment:
That is a nice pick.It was deprecated in IOS 10 [link](http://stackoverflow.com/questions/38964264/openurl-in-ios10). Currently i am handicapped due to lack of resources for testing ios part.But soon i will get in touch with my friends who own resources so that i can utilise them to amend changes.